### ISSUE TITLE
android: FATAL EXCEPTION: heightChangeTimer

### ISSUE THREAD

**Issue by a-eid** [mateusz1913/react-native-avoid-softinput#62]
### Environment

Library version: ^2.4.3
OS version: android android 10

### Affected platforms

- [X] Android
- [ ] iOS

### Current behavior

when navigating while the keyboard is open, the app crashes 
also it does happen randomly sometimes.

crash 

```bash
FATAL EXCEPTION: heightChangeTimer

kotlin.TypeCastException: null cannot be cast to non-null type com.facebook.react.uimanager.RootView

FATAL EXCEPTION: heightChangeTimer
Process: com.zeal.dara, PID: 26493
kotlin.TypeCastException: null cannot be cast to non-null type com.facebook.react.uimanager.RootView
	at com.reactnativeavoidsoftinput.AvoidSoftInputManager.onSoftInputHeightChange(AvoidSoftInputManager.kt:125)
	at com.reactnativeavoidsoftinput.AvoidSoftInputManager.onSoftInputHeightChange$default(AvoidSoftInputManager.kt:104)
	at com.reactnativeavoidsoftinput.AvoidSoftInputModule.onSoftInputHeightChange(AvoidSoftInputModule.kt:114)
	at com.reactnativeavoidsoftinput.AvoidSoftInputProvider$onSoftInputHeightChange$$inlined$schedule$1.run(Timer.kt:149)
	at java.util.TimerThread.mainLoop(Timer.java:562)
	at java.util.TimerThread.run(Timer.java:512
```

### Expected behavior

for it to work without crashing.		

### Reproduction

```tsx
  const footerPaddingValue = useSharedValue(0)

  const onFocusEffect = React.useCallback(() => {
    AvoidSoftInput.setAdjustNothing()
    AvoidSoftInput.setEnabled(true)

    return () => {
      AvoidSoftInput.setEnabled(false)
      AvoidSoftInput.setDefaultAppSoftInputMode()
    }
  }, [])

  // @ts-ignore
  onFocusEffect(onFocusEffect)

  useSoftInputHeightChanged(({softInputHeight}) => {
    footerPaddingValue.value = withTiming(softInputHeight)
  })

  const style = useAnimatedStyle(
    () => ({
      height: WINDOW.height - footerPaddingValue.value,
    }),
    [],
  )

  return (
    <A.View style={style}>
    
	 <TextInput onChangeText={() => {
	      navigation.dispatch(CommonActions.reset({ })}} 
    />    
    
    </A.View>
  )

```

**Comment by a-eid** [comment]
```ts
  const style = useAnimatedStyle(
    () => ({
      flex:1,
      paddingBottom:  footerPaddingValue.value,
    }),
    [],
  )
```

also crashes with the same error. 

the only workaround is to dismiss the keyboard before navigating.

**Comment by batical** [comment]
Same here, using animating style but the app is crashing when clicking on input so showing the keyboard

**Comment by mateusz1913** [comment]
closed with #64

**Comment by mateusz1913** [comment]
Available with v2.4.4 🎉 , thanks @batical !
